dspace-server-webapp/src/test/java/org/dspace/app/rest/matcher/SubmissionFormFieldMatcher.java

```diff
@@ -113,19 +116,26 @@ private SubmissionFormFieldMatcher() {
      *            the optional filter to be used for the lookup
      * @param searchConfiguration
      *            the searchConfiguration to be used for the lookup
+     * @param nameVariants
+     *            the optional name variants allowed flag
      * @return a Matcher for all the condition above
      */
     public static Matcher<? super Object> matchFormOpenRelationshipFieldDefinition(String type, String label,
                                                                                    String mandatoryMessage,
-                                                                                   boolean repeatable, String hints,
+                                                                                   boolean repeatable,
+                                                                                   String hints,
                                                                                    String metadata,
                                                                                    String relationshipType,
                                                                                    String filter,
-                                                                                   String searchConfiguration) {
+                                                                                   String searchConfiguration,
+                                                                                   @Nullable Boolean nameVariants) {
```

Comment:
To be honest, I'm not sure I understand why we are using `@Nullable` here?  I thought this annotation simply says "this field can be null at times".  But a `Boolean` can already be `null`, so the usage here is unclear to me.

Reply:
This has been adjusted